cheddar/cheddar-persistence/src/main/java/com/clicktravel/cheddar/infrastructure/persistence/database/configuration/ItemConfiguration.java

```diff
@@ -227,4 +227,9 @@ public String indexNameForQuery(final AttributeQuery attributeQuery) {
     private String compoundName(final String propertyName, final String supportingPropertyName) {
         return propertyName + "_" + supportingPropertyName;
     }
+
```

Comment:
I reckon a small bit of java doc here to further confirm what this is for would help. The name is pretty descriptive but still very confusing. Perhaps a simple example of what this is try to combat might help other not be bamboozled by it.